components/List.tsx

```diff
@@ -0,0 +1,14 @@
+interface Props {
+  lists: { content: string }[];
+}
+export function List({ lists }: Props) {
+  const className = "grid grid-cols-1 gap-1 list-decimal";
+  return (
+    <ul className={className}>
+      {lists.map((list, idx) => {
+        const { content } = list;
```

Comment:
**praise:** We should use destructuring directly in the map and return pure component (e.g. no `{ return ... }`). Example:

`lists.map(({content}) => <li>{content}</li>)`